openreview/tools.py

```diff
@@ -1716,10 +1716,14 @@ def get_conflicts(author_profiles, user_profile, policy='default', n_years=5):
 
     conflicts = set()
     conflicts.update(author_domains.intersection(user_info['domains']))
-    conflicts.update(author_relations.intersection(user_info['emails']))
-    conflicts.update(author_emails.intersection(user_info['relations']))
-    conflicts.update(author_emails.intersection(user_info['emails']))
-    conflicts.update(author_publications.intersection(user_info['publications']))
+    if not conflicts:
```

Comment:
why do you add an if here? what does it change?

Reply:
sometimes we want to have the full list of conflicts for debugging purposes.

Reply:
Reverted.